packages/ember-template-compiler/lib/system/compile-options.js

```diff
@@ -10,6 +10,7 @@ export default function compileOptions(_options) {
   if (options.moduleName) {
     let meta = options.meta;
     meta.moduleName = options.moduleName;
+    meta.managerId = options.managerId;
```

Comment:
This block is only meant as a fallback for older versions of ember-cli-htmlbars which passed `{ moduleName: 'here' }` instead of `{ meta: { moduleName: 'here' } }`.  No need to put managerId here like this.